packages/amplication-data-service-generator/src/create-data-service-impl.ts

```diff
@@ -128,7 +132,13 @@ function prepareEntityPluralName(entities: Entity[]): Entity[] {
   return currentEntities;
 }
 
-function resolveLookupFields(entities: Entity[]): Entity[] {
+function resolveLookupFields(
+  entities: Entity[],
+  installedPlugins: Plugin[]
+): Entity[] {
+  const isMySQLPluginInstalled = installedPlugins.find(
+    (plugin) => plugin.pluginId === MYSQL_PLUGIN_ID
+  );
   const entityIdToEntity: Record<string, Entity> = {};
```

Comment:
this validation should be part of the plugin and not part of DSG
The plugin should handle one of the main events (e.g. beforeCreateServer) and if needed the plugin will:
1. write the error message to the logger 
2. throw and error? or "skip" the entire process